network-api/networkapi/templates/fragments/libraries/filter_form.html

```diff
@@ -7,8 +7,15 @@
                     <h3 class="tw-h4-heading">{{ field.label_tag }}</h3>
                 </legend>
                 {{ field.errors }}
-                <ul class="tw-list-none tw-mb-0 tw-pl-0 {% if field|length > 5 %} tw-max-h-[13.5rem] tw-overflow-y-scroll{% endif %}">
-                    {% for option in field %}<li class="tw-flex tw-flex-row tw-items-baseline">{{ option }}</li>{% endfor %}
```

Comment:
This was rendering the `input` tags inside the `label` tags, which made the CSS look off and introduced the bug. The new form rendering fixes that.